lib/__tests__/standalone-cache.test.js

```diff
@@ -223,6 +223,91 @@ describe('standalone cache uses cacheLocation', () => {
 	});
 });
 
+describe('standalone cache (enabled in config)', () => {
+	const cwd = path.join(__dirname, 'tmp', 'standalone-cache');
+
+	safeChdir(cwd);
+
+	const expectedCacheFilePath = path.join(cwd, '.stylelintcache');
+
+	function getConfigWithCache(additional = {}) {
+		return {
+			files: replaceBackslashes(path.join(fixturesPath, 'cache', '*.css')),
+			config: {
+				cache: true,
+				rules: { 'block-no-empty': true, 'color-no-invalid-hex': true },
+			},
+			...additional,
+		};
+	}
+
+	beforeEach(async () => {
+		// Initial run to warm up the cache
+		await standalone(getConfigWithCache());
+	});
+
+	afterEach(async () => {
+		// Clean up after each test case
+		await removeFile(expectedCacheFilePath);
+		await removeFile(newFileDest);
+	});
+
+	it('uses cache when no CLI argument is provided', async () => {
```

Comment:
[suggest] I think relating `standalone()` with CLI may be confusing since `standalone()` is called not only CLI but also Node.js programs. So, can you rewrite the descriptions around CLI or `--cache`?